components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -598,7 +598,7 @@ sealed class WebExtensionAction : BrowserAction() {
         WebExtensionAction()
 
     /**
-     * Updates the [WebExtensionState.enabled] flag.
+     * Updates the [WebExtensionState.allowedInPrivateBrowsing] flag.
```

Comment:
`allowedInPrivateBrowsing`? 🤔

Reply:
Yes, the docs were wrong (copy/paste mistake). I've put it in a separate commit now.